triggers/triggers-dt/src/main/java/gg/xp/xivsupport/triggers/Arcadion/M1N.java

```diff
@@ -0,0 +1,119 @@
+package gg.xp.xivsupport.triggers.Arcadion;
+
+import gg.xp.reevent.events.BaseEvent;
+import gg.xp.reevent.events.EventContext;
+import gg.xp.reevent.scan.AutoChildEventHandler;
+import gg.xp.reevent.scan.AutoFeed;
+import gg.xp.reevent.scan.FilteredEventHandler;
+import gg.xp.xivdata.data.duties.*;
+import gg.xp.xivsupport.callouts.CalloutRepo;
+import gg.xp.xivsupport.callouts.ModifiableCallout;
+import gg.xp.xivsupport.events.actlines.events.AbilityCastStart;
+import gg.xp.xivsupport.events.actlines.events.AbilityUsedEvent;
+import gg.xp.xivsupport.events.actlines.events.SnapshotLocationDataEvent;
+import gg.xp.xivsupport.events.state.XivState;
+import gg.xp.xivsupport.events.triggers.seq.EventCollector;
+import gg.xp.xivsupport.events.triggers.seq.SequentialTrigger;
+import gg.xp.xivsupport.events.triggers.seq.SqtTemplates;
+import gg.xp.xivsupport.events.triggers.support.NpcCastCallout;
+import gg.xp.xivsupport.models.Position;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+@CalloutRepo(name = "M1N", duty = KnownDuty.M1N)
+public class M1N extends AutoChildEventHandler implements FilteredEventHandler {
+	public static final Logger log = LoggerFactory.getLogger(M1N.class);
+
+	@NpcCastCallout(0x9309)
+	private final ModifiableCallout<AbilityCastStart> oneTwoPawEastWest = new ModifiableCallout<>("One-Two Paw: West safe", "West then East");
```

Comment:
If the cast times for these abilities are accurate (i.e. the important part of the ability happens at the end of the cast bar), you can use ModifiableCallout.durationBasedCall to automatically have the remaining time in the on-screen text.